host/xtest/xtest_main.c

```diff
@@ -166,6 +166,10 @@ int main(int argc, char *argv[])
 #ifdef CFG_SECURE_DATA_PATH
 	else if (argc > 1 && !strcmp(argv[1], "--sdp-basic"))
 		return sdp_basic_runner_cmd_parser(argc-1, &argv[1]);
+#endif
+#ifdef CFG_PKCS11_TA
+	else if (argc > 1 && !strcmp(argv[1], "--pkcs11_1028_destroy_token_object"))
```

Comment:
argc == 2?

Reply:
+1 for `pkcs11-1028-destroy-token-object`